ledger-core/virtual/handlers/vdelegatedrequestfinished.go

```diff
@@ -134,6 +140,7 @@ func (s *SMVDelegatedRequestFinished) stepProcess(ctx smachine.ExecutionContext)
 		}
 
 		s.updateSharedState(ctx, state)
+		s.updateMemoryCache(ctx, state)
```

Comment:
Ehm, you make async call request from there. Are you really need to do it from that function?

Reply:
fixed